eclipse-scout-core/src/form/fields/smartfield/ProposalField.js

```diff
@@ -189,11 +189,12 @@ export default class ProposalField extends SmartField {
     // this causes a lookup which may fail and open a new proposal chooser (property
     // change for 'result').
     if (searchTextChanged) {
+      this.clearErrorStatus();
```

Comment:
Code changes look Ok, but please add a Jasmine (ProposalFieldSpec.js) or a Selenium (com.bsiag.widgets.ui.html.form.fields.smartfield.SmartFieldTest) test for the reported issue and bugfix. Without it, someone could easily break your bugfix with a later change without noticing.

Also fix the commit message. We only add the plain ticket-no. on the last line, without further explanation or prefixes.